src/migrator.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import Command from 'cli-engine-command'
+import Plugins from './plugins'
+import path from 'path'
+import fs from 'fs-extra'
+
+export default class extends Command {
+  plugins = new Plugins(this)
+
+  async run () {
+    let file = path.join(this.config.dirs.data, '..', 'heroku', 'plugins', 'plugins.json')
+    let pljson
+    try {
+      pljson = fs.readJSONSync(file)
+    } catch (err) {
+      this.debug(err.message)
+      return
+    }
+    for (let p of pljson) {
+      try {
+        // this.log('migrating V5 plugins')
+        if (p.tag === 'symlink') {
+          // this.log('install via symlink: ', p.name)
+          let pluginPath = fs.realpathSync(path.join(this.config.dirs.data, '..', 'heroku', 'plugins', 'node_modules', p.name))
+          await this.plugins.addLinkedPlugin(pluginPath)
+        } else {
+          // this.log('install via npm: ', p.name)
+          await this.plugins.install(p.name)
+        }
+      } catch (err) {
+        this.debug(err.message)
```

Comment:
@RasPhilCo I would warn here so that people know that their plugins / commands are missing.  In most cases they will not see it since I think this will be run in the background (have not been in this code much yet) but for beta testers, having a warn here with the err would be valuable.